wca-regulations.md

```diff
@@ -521,5 +521,8 @@ Organization teams may adopt optional regulations to facilitate the administrati
 - Z1) The organization team may require competitors to submit puzzles during registration.
 - Z2) The organization team may limit the number of events per competitor.
 - Z3) The organization team may select competitors who directly qualify for certain rounds of certain events based on the results of specific previous competitions.
-- Z4) The organization team may limit the number of competitors per event or per competition, on either a "first come first serve" basis or based upon qualification results or rankings in the WCA world rankings of a previously announced calendar date.
+- Z4) The organization team may limit the number of competitors per event or per competition, on either a "first come first serve" basis or based upon qualification results or rankings in the WCA world rankings of a previously announced calendar date. If the number of competitors per event or per competition is limited, the following regulations apply:
+    - Z4a) If the limit for a competition or an event is reached before the competition, the organization team must not accept on-site registrations for the competition or the event.
+    - Z4b) All limits must be approved by the WCA Board and may only be changed for legitimate reasons (e.g. when the venue was changed).
```

Comment:
"legitimate" is wishy-washy here, since it really just means "whatever the WCA board accept.

How about:
1) Competitor limits should be announced together with the competition. Any changes to competitor limits are at the discretion of the WCA Board.

I would put the example in the Guidelines, along with one or two other reasonable ones.  (What are common reasons that are accepted? If there aren't any other, can we require venue change as the reason?)

Reply:
From what I know the most common reasons are venue changes and/or much more people registering than expected. I have added a guideline for these two examples.